examples/sdk/auth-social/src/pages/Register.tsx

```diff
@@ -0,0 +1,84 @@
+import { WebAuthnSigner } from '@dfns/sdk-browser'
+import React from 'react'
+import { Link } from 'react-router-dom'
+
+import { CredentialResponse, GoogleLogin } from '@react-oauth/google';
+
+import '../globals.css'
+import { dfnsApi, setAuthToken } from '../api'
+import { useAppContext } from '../hooks/useAppContext';
+
+export default function Register(): JSX.Element {
```

Comment:
nit: these should be `React.JSX.Element`right? Appears repeatedly in the code... Also, I don't think you should need a return type if react is correctly setup unless you want to specify a certain type of component...